tests/application/admin/test_views.py

```diff
@@ -220,8 +220,8 @@ def test_admin_user_can_share_page_with_dept_user(test_app_client, mock_dept_use
     resp = test_app_client.get(
         url_for(
             "static_site.measure_page",
-            topic_slug=stub_measure_page.parent.parent.slug,
-            subtopic_slug=stub_measure_page.parent.slug,
+            topic_slug=stub_measure_page.measure.subtopic.topic.slug,
```

Comment:
Should this be `stub_measure_version`?

Reply:
Probably, but as mentioned in PR summary, we have a separate ticket to "fix all the names" (https://trello.com/c/WFbfRGEX/1283-rename-things-after-page-table-refactor-for-consistency)